app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -612,27 +618,56 @@ class Identification extends React.Component {
 }
 
 const schema = yup.object().shape({
-  first_name: yup.string().required('Please enter a First Name.').max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
-  middle_name: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
-  last_name: yup.string().required('Please enter a Last Name.').max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
+  first_name: yup
```

Comment:
these should be able to fit on one line, please run eslint

Reply:
👍

Reply:
https://github.com/SaraAlert/SaraAlert/blob/985cc43322dd9e8743a0b6e5935220a2b1155f90/app/javascript/components/enrollment/steps/Identification.js#L620